src/main/java/edu/hm/hafner/analysis/parser/CodeCheckerParser.java

```diff
@@ -21,7 +21,7 @@
 public class CodeCheckerParser extends LookaheadParser {
     private static final long serialVersionUID = -3015592762345283582L;
     private static final String CODE_CHECKER_DEFECT_PATTERN =
-            "^\\[(?<severity>CRITICAL|HIGH|MEDIUM|LOW)\\] (?<path>.+):(?<line>\\d+):(?<column>\\d+): (?<message>.*?) \\[(?<category>[^\\s]*?)\\]$";
+            "^\\[(?<severity>UNSPECIFIED|STYLE|CRITICAL|HIGH|MEDIUM|LOW)\\] (?<path>.+):(?<line>\\d+):(?<column>\\d+): (?<message>.*?) \\[(?<category>[^\\s]*?)\\]$";
```

Comment:
I know it makes no difference for the regular expression, but wouldn't it be nice for the reader, to see the severity levels in their order?

Reply:
These are unordered thou. Ie we have no knowledge of their severity.

Reply:
I got that. I just didnt think it mattered to much. I can change.